Source/Scene/Cesium3DTileset.js

```diff
@@ -70,19 +72,25 @@ define([
         }
         //>>includeEnd('debug');
 
-        var tilesJson;
+        var tilesetUrl;
+        var baseUrl;
+
         if (getExtensionFromUri(url) === 'json') {
-            tilesJson = url;
-            url = getBaseUri(url);
+            tilesetUrl = url;
+            baseUrl = getBaseUri(url);
+        } else if (isDataUri(url)) {
+            tilesetUrl = url;
+            baseUrl = '';
         } else {
-            url = appendForwardSlash(url);
-            tilesJson = url + 'tiles.json';
+            baseUrl = appendForwardSlash(url);
+            tilesetUrl = baseUrl + 'tiles.json';
         }
 
         this._url = url;
```

Comment:
Originally I was worried about 
```javascript
collectionOptions.url = getAbsoluteUri(gltfUrl, this._tileset.url);
```
in `Instanced3DModel3DTileContentProvider` but it's not a problem anymore now that we use `getAbsoluteUri`